login-workflow/src/new-architecture/screens/SuccessScreen/ExistingAccountSuccessScreen.tsx

```diff
@@ -0,0 +1,39 @@
+import { Person } from '@mui/icons-material';
+import React from 'react';
+import { useRegistrationContext } from '../../contexts';
+import { useLanguageLocale } from '../../hooks';
+import { SuccessScreenBase } from './SuccessScreenBase';
+import { SuccessScreenProps } from './types';
+
+export const ExistingAccountSuccessScreen: React.FC<SuccessScreenProps> = (props) => {
```

Comment:
Please add params here. I know, we are missing those in some screens, but it is good time to do that.